goopenssl.h

```diff
@@ -105,6 +106,48 @@ go_openssl_EVP_CipherUpdate_wrapper(GO_EVP_CIPHER_CTX_PTR ctx, unsigned char *ou
     return go_openssl_EVP_CipherUpdate(ctx, out, &len, in, in_len);
 }
 
+static inline int
+go_openssl_EVP_PKEY_get_raw_public_key_wrapper(const GO_EVP_PKEY_PTR pkey, unsigned char *pub, size_t len)
```

Comment:
These wrappers introduce a potential bug. When `len` is copied to the C stack we take the address of it, but we only have a reference to the value on the C stack, so the calling Go code does not receive the updated value, should it change.